dashboard/app/controllers/foorm/simple_survey_controller.rb

```diff
@@ -1,24 +1,24 @@
 module Foorm
-  class MiscSurveyController < ApplicationController
-    MISC_FOORM_SUBMIT_API = '/dashboardapi/v1/foorm/misc_survey_submission'
+  class SimpleSurveyController < ApplicationController
+    FOORM_SIMPLE_SURVEY_SUBMIT_API = '/dashboardapi/v1/foorm/simple_survey_submission'
 
-    # General misc survey.
+    # General simple survey.
     # GET '/form/:misc_form_path'
```

Comment:
Note this `misc_form_path` is still around a lot, since we use that field to get the right configuration from the list in `MiscSurvey` (now `SimpleSurveySubmission`). I will rename all of these references (to `SimpleSurveyForm.path`) once we move over to use the new `SimpleSurveyForm` model to figure out what Form to show.

Reply:
Depending on how far out this work is, it might be worth adding a comment about that. (If it's soon, no need)

Reply:
Sorry, should have clarified -- working on it immediately. Just trying to break up the PRs a bit.